join-us.md

```diff
@@ -0,0 +1,21 @@
+# Presentation
+
+**_Design all the things_** is an open initiative imagined by [Sylvain Fraïssé](https://github.com/fraisse) and [Jérôme Avoustin](https://github.com/rehia) to explore and learn how a "true" _Domain Driven Design_ approach could help us write better softwares. The purpose is not to build a product, but to learn new things, and make our mental models evolve.
+
+It is organized around face-to-face workshops, and probably remote as well. They are open to anyone who wants to contribute. You can come and contribute to the whole project, or just come once. We will learn together. If you want to know more, join the [_Montpellier Communautés_ slack](https://bit.ly/comm-mtp), and follow the #ddd channel. We organize meetups on it.
```

Comment:
and follow the #ddd channel -> and follow the #ddd channel (`design all the things` keyword).